src/Microsoft.Data/Microsoft.Data.DataFrame.csproj

```diff
@@ -69,46 +94,74 @@
     </Compile>
-    <Compile Update="DataFrameBinaryOperators.cs">
+    <Compile Update="DataFrame.BinaryOperations.cs">
       <DesignTime>True</DesignTime>
       <AutoGen>True</AutoGen>
-      <DependentUpon>DataFrameBinaryOperators.tt</DependentUpon>
+      <DependentUpon>DataFrame.BinaryOperations.tt</DependentUpon>
     </Compile>
-    <Compile Update="DataFrameColumnArithmeticTemplate.cs">
+    <Compile Update="DataFrame.BinaryOperators.cs">
       <DesignTime>True</DesignTime>
       <AutoGen>True</AutoGen>
-      <DependentUpon>DataFrameColumnArithmeticTemplate.ttinclude</DependentUpon>
+      <DependentUpon>DataFrame.BinaryOperators.tt</DependentUpon>
+    </Compile>
+    <Compile Update="DataFrameBinaryOperators.cs">
+      <DesignTime>True</DesignTime>
+      <AutoGen>True</AutoGen>
+      <DependentUpon>DataFrameBinaryOperators.tt</DependentUpon>
     </Compile>
     <Compile Update="PrimitiveColumn.BinaryOperations.cs">
       <DesignTime>True</DesignTime>
       <AutoGen>True</AutoGen>
       <DependentUpon>PrimitiveColumn.BinaryOperations.tt</DependentUpon>
     </Compile>
+    <Compile Update="PrimitiveColumn.Computations.cs">
+      <DesignTime>True</DesignTime>
+      <AutoGen>True</AutoGen>
+      <DependentUpon>PrimitiveColumn.Computations.tt</DependentUpon>
+    </Compile>
     <Compile Update="PrimitiveColumnArithmetic.cs">
       <DesignTime>True</DesignTime>
       <AutoGen>True</AutoGen>
       <DependentUpon>PrimitiveColumnArithmetic.tt</DependentUpon>
     </Compile>
+    <Compile Update="PrimitiveColumnComputations.cs">
+      <DesignTime>True</DesignTime>
+      <AutoGen>True</AutoGen>
+      <DependentUpon>PrimitiveColumnComputations.tt</DependentUpon>
+    </Compile>
     <Compile Update="PrimitiveColumnContainer.BinaryOperations.cs">
       <DesignTime>True</DesignTime>
       <AutoGen>True</AutoGen>
       <DependentUpon>PrimitiveColumnContainer.BinaryOperations.tt</DependentUpon>
     </Compile>
-    <Compile Update="PrimitiveDataFrameColumnArithmetic.cs">
+    <Compile Update="PrimitiveColumnArithmetic.cs">
+      <DesignTime>True</DesignTime>
+      <AutoGen>True</AutoGen>
+      <DependentUpon>PrimitiveColumnArithmetic.tt</DependentUpon>
+    </Compile>
+    <Compile Update="PrimitiveColumnContainer.BinaryOperations.cs">
       <DesignTime>True</DesignTime>
       <AutoGen>True</AutoGen>
-      <DependentUpon>PrimitiveDataFrameColumnArithmetic.tt</DependentUpon>
+      <DependentUpon>PrimitiveColumnContainer.BinaryOperations.tt</DependentUpon>
     </Compile>
-    <Compile Update="PrimitiveDataFrameColumnContainer.BinaryOperations.cs">
+    <Compile Update="Strings.Designer.cs">
       <DesignTime>True</DesignTime>
       <AutoGen>True</AutoGen>
-      <DependentUpon>PrimitiveDataFrameColumnContainer.BinaryOperations.tt</DependentUpon>
+      <DependentUpon>Strings.resx</DependentUpon>
     </Compile>
   </ItemGroup>
 
+  <ItemGroup>
+    <EmbeddedResource Update="Strings.resx">
```

Comment:
I think the casing needs to match the actual file casing - is it possible to rename the file?

See [https://stackoverflow.com/questions/1793735/change-case-of-a-file-on-windows](https://stackoverflow.com/questions/1793735/change-case-of-a-file-on-windows) for how to do this on Windows.